test/lsp/ui/vim/completion.vimspec

```diff
@@ -0,0 +1,37 @@
+Describe lsp#uivim#completion
+
+  Before each
+    %delete _
+    setlocal filetype=html
+    setlocal omnifunc=lsp#omni#complete
+  End
+
+  It should expand simple snippet with multibyte chars
+    Skip This test needs asynchronous process and snippetSupport=true
```

Comment:
Should we create a `wait(timeout)` function? Something like how we do in asyncwait. https://github.com/prabirshrestha/async.vim/blob/master/autoload/async/job.vim#L240-L253

then we can actually run the test in ci.

If we do want to fix vim-themis I pasted some notes at https://github.com/thinca/vim-themis/issues/61

Reply:
`while-sleep` is not interrupted other any process in vim e.g. feedkeys/timer etc.
So we want to `done` feature like `mocha`, I think.

```viml
It example test
  let ctx = {}
  let ctx.done = themis#async()
  let ctx.changedtick = getbufvar('%', '&changedtick')
  function! ctx.callback(timer)
    if self.changedtick == getbufvar('%', '&changedtick')
      return
    endif
    " some tests...
    call self.done()
  endfunction
  call some#async#job()
  call timer_start(100, { timer -> ctx.callback(timer) }, { 'repeat': -1 })
End
```

Reply:
It is good that you still have the tests that is skipped so at least we can manually verify in case we change anything in completion engine in future